src/Types.jl

```diff
@@ -1363,7 +1363,7 @@ function find_registered!(env::EnvCache,
         end
     end
     # if there's still nothing to look for, return early
-    isempty(names) && isempty(uuids) && return
+    !force && isempty(names) && isempty(uuids) && return
```

Comment:
I am not 100% sure on this one, but the same check happens on L1344 above, and I think the `force` option is meant to go past this here and read the registry?

Reply:
It seems like pulling out the `force` option entirely might be best.